### PR TITLE
Add Hashtag counter controller and app

### PR DESCRIPTION
**Pull request by youngbupark**
* Add HashTag Counter controller
* Add hashtag counter app to k8s deployment

Issue: #14

### CODE REVIEW

File: hashtag-app/Controllers/HashTagController.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        private readonly ILogger<HashTagController> logger;
+        private readonly IConfiguration configuration;
+
+        public HashTagController(ILogger<HashTagController> logger, IConfiguration config)
+        {
+            this.logger = logger;
+            this.configuration = config;
+        }
+
+        [HttpPost("message_binding")]
```
Comment (LMWF) [inline hashtag-app/Controllers/HashTagController.cs]: `messagebinding`

File: hashtag-app/Controllers/HashTagController.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            var proxy = ActorProxy.Create<IHashTagActor>(actorId, this.configuration[AppSettings.HashTagCounterActorType]);
+
+            this.logger.LogTrace($"Increase ${message.Sentiment}.");
+            try
+            {
+                await proxy.Increment(message.Sentiment);
+            }
+            catch (Exception e)
+            {
+                this.logger.LogError(e, "message-topic");
```
Comment (LMWF) [inline hashtag-app/Controllers/HashTagController.cs]: What's message-topic here?

Comment (youngbupark) [reply hashtag-app/Controllers/HashTagController.cs]: rename this as well :)

File: hashtag-app/Controllers/HashTagController.cs (hunk 1 above)
Comment (youngbupark) [reply hashtag-app/Controllers/HashTagController.cs]: done

**Pull request merged**